docs/csharp/roslyn-sdk/snippets/source-generators/ConsoleApp/ConsoleApp.csproj

```
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <LangVersion>11.0</LangVersion>
```

This is already the default for `net7.0`, I think we can remove the `LangVersion` but we should add `EmitCompilerGeneratedFiles`.

```suggestion
    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
```